udata/core/dataset/apiv2.py

```diff
@@ -265,9 +265,9 @@ class DatasetSearchAPI(API):
     @apiv2.marshal_with(dataset_page_fields)
     def get(self):
         """List or search all datasets"""
-        search_parser.parse_args()
+        args = search_parser.parse_args()
         try:
-            return search.query(Dataset, **multi_to_dict(request.args))
+            return search.query(Dataset, **args)
```

Comment:
Makes more sense indeed.

Note for self: we have the same pattern elsewhere 
- https://github.com/opendatateam/udata/blob/bfeae1b6a1d0d33c29485a2feae012db69eec38f/udata/core/reuse/apiv2.py#L29-L30
- https://github.com/opendatateam/udata/blob/bfeae1b6a1d0d33c29485a2feae012db69eec38f/udata/core/organization/apiv2.py#L34-L35
- https://github.com/opendatateam/udata/blob/bfeae1b6a1d0d33c29485a2feae012db69eec38f/udata/core/dataservices/apiv2.py#L28-L29